packages/utils/node-resolver-core/src/NodeResolver.js

```diff
@@ -49,6 +52,26 @@ type Module = {|
   code?: string,
 |};
 
+const parseFilePath = (
```

Comment:
Wonder which parts of specifier parsing should be up to the resolver and which should be up to core. I guess the pipeline is parsed by core. Should query params be parsed by core as well? No strong opinion just wondering.

Reply:
Probably?